contracts/optimistic-ethereum/OVM/bridge/OVM_L2CrossDomainMessenger.sol

```diff
@@ -17,9 +17,12 @@ import { OVM_BaseCrossDomainMessenger } from "./OVM_BaseCrossDomainMessenger.sol
 
 /**
  * @title OVM_L2CrossDomainMessenger
- * @dev L2 CONTRACT (COMPILED)
- * This contract lives on L2. It sends messages to L1, and relays them from L1.
- */
+ * @dev The L2 Cross Domain Messenger contract sends messages from L2 to L1, and relays messages from L1 onto L1.
```

Comment:
```suggestion
 * @dev The L2 Cross Domain Messenger contract sends messages from L2 to L1, and is the entry point for L2 messages sent via the L1 Cross Domain Messenger.
```
What I'm trying to get at here is that the relaying is passive, and that `enqueue` can be used to send L1->L2 messages which bypass this contract, so it is not *always* relaying the messages.